coriolis/utils.py

```diff
@@ -276,7 +276,7 @@ def write_winrm_file(conn, remote_path, content, overwrite=True):
             data = data.encode()
         asb64 = base64.b64encode(data).decode()
         cmd = ("$ErrorActionPreference = 'Stop';"
-               "$x = [System.IO.FileStream]::new(\"%s\", "
+               "$x = New-Object System.IO.FileStream(\"%s\", "
```

Comment:
Out of curiosity, is there any behavioral difference for this change?

Reply:
This didn't work on Powershell 4.0. We have PSv4 on the WS2012R2 image from our site.